controllers/course.js

```diff
@@ -0,0 +1,71 @@
+const { Types } = require("mongoose");
+const { ObjectId } = Types;
+const {
+  getOne,
+  getAll,
+  updateDocument,
+  deleteDocument,
+  createDocument,
+} = require("../utils/crudOperations");
+const { CourseExcludedFields } = require("../utils/excludedFields");
```

Comment:
`CourseExcludedFields ` this seems not to be defined, please check and be sure that it is defined